code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,293 @@
+			new /obj/item/dnainjector/glow(src)
+			new /obj/item/dnainjector/spastic(src)
+			new /obj/item/dnainjector/epimut(src)
+			new /obj/item/handmirror(src)
+			new /obj/item/razor(src)
+		if(134 to 138)
+			for(var/stick in 7)
+				stick = pick(subtypesof(/obj/item/flashlight/glowstick))
+				new stick(src)
+		if(139 to 141)
+			new /obj/effect/spawner/lootdrop/two_percent_xeno_egg_spawner(src)//Two percent for a crate to spawn a two percent xeno spawner.
+			new /obj/item/clothing/suit/xenos(src)                            //This means it's about one percent or less to get a xeno egg from an abandoned crate, I don't know.
+			new /obj/item/toy/toy_xeno(src)                                   //I actually don't give a shit how much percent it has.
+			new /obj/item/clothing/head/xenos(src)                            //I'm not memorizing the multiplication table. I don't have a calculator and my ass is fat.
+		if(142 to 146)
+			new /mob/living/simple_animal/cluwne(src)
+			new /mob/living/simple_animal/cluwne(src)
+			new /mob/living/simple_animal/cluwne(src)
+			new /mob/living/simple_animal/cluwne(src)
+			new /obj/item/dnainjector/clumsymut(src)
+			new /obj/item/dnainjector/cluwnemut(src)
+		if(147)
+			new /obj/item/clothing/head/helmet/swat/nanotrasen(src)
+			new /obj/item/clothing/under/rank/security/officer(src)
+			new /obj/item/clothing/suit/armor/vest(src)
+			new /obj/item/clothing/shoes/combat/swat(src)
+			new /obj/item/clothing/gloves/combat(src)
+		if(148)
+			for(var/ears in 4)
+				ears = pick(subtypesof(/obj/item/organ/ears))
+				new ears
+		if(149)
+			for(var/soda in 8)
+				soda = pick(subtypesof(/obj/item/reagent_containers/food/drinks/soda_cans) - /obj/item/reagent_containers/food/drinks/soda_cans)
+				new soda(src)
+		if(150)
+			new /obj/item/bedsheet/centcom(src)
+			new /obj/item/clothing/under/rank/centcom/commander(src)
+			new /obj/item/card/id/centcom(src)
+			new /obj/item/radio/headset/headset_cent/alt(src)
+			new /obj/item/clothing/head/centhat(src)
+			new /obj/effect/decal/remains/human(src)
+			new /obj/item/gun/energy/e_gun(src)
+			new /obj/item/paper/fluff/awaymissions/centcom/gateway_memo(src)
+		if(151)
+			new /obj/item/reagent_containers/glass/bottle/felinid(src)
+		if(152 to 155)
+			for(var/backpack in 1 to rand(3, 8))
+				backpack = pick(subtypesof(/obj/item/storage/backpack))
+				new backpack(src)
+		if(156)
+			new /obj/item/clothing/head/chefhat(src)
+			new /obj/item/grenade/chem_grenade/teargas/moustache(src)
+			new /obj/item/grenade/chem_grenade/teargas/moustache(src)
+		if(157)
+			new /obj/item/storage/belt/janitor/full(src)
+			new /obj/item/reagent_containers/spray/chemsprayer/janitor(src)
+			new /obj/item/clothing/suit/bio_suit/janitor(src)
+			new /obj/item/clothing/head/bio_hood/janitor(src)
+			new /obj/item/clothing/under/rank/civilian/janitor(src)
+			new /obj/item/toy/gun(src)
+		if(158)
+			new /obj/item/golem_shell/servant(src)
+			new /obj/item/golem_shell/servant(src)
```

Comment:
Wont these just create ghost roles and have live golems spawning inside the crate while its locked?

Reply:
it's incomplete shell